backend/PennCourses/docs_settings.py

```diff
@@ -866,53 +866,86 @@ def _get_responses(self, path, method):
         else:
             response_schema = item_schema
 
+        # Automatically add 403 response if authentication is required
+        auth_response_dict = (
+            {403: {"description": "Authentication credentials were not provided."}}
+            if IsAuthenticated in self.view.permission_classes
+            else {}
+        )
+
         if (
             path not in self.response_codes.keys()
             or method not in self.response_codes[path].keys()
             or not self.response_codes[path][method]
         ):
             if path in self.override_schema and method in self.override_schema[path].keys():
                 return {
+                    **{
+                        "200": {
+                            "description": "",
+                            **(
+                                {
+                                    "content": {
+                                        ct: {"schema": self.override_schema[path][method][200]}
```

Comment:
I'm guilty of this too, but could you spell out what this variable is? one and two letter variable names can get confusing.

Reply:
looks like it's `content_type`

Reply:
This naming convention was from the DRF AutoSchema code but you're definitely right that `content_type` is better.